extractors/extractors.go

```diff
@@ -47,7 +47,7 @@ func Extract(u string, option Options) ([]*Data, error) {
 	}
 	extractor := extractorMap[domain]
 	if extractor == nil {
-		extractor = extractorMap[""]
+		return nil, errors.Errorf("%s is not yet supported", u)
```

Comment:
This actually refers to the universal extractor:

https://github.com/iawia002/lux/blob/b3422e84877a04f3fd4ac4e3a07b697ec44e0e9d/extractors/universal/universal.go#L11-L13

Maybe we should update the name to avoid misunderstandings

Reply:
Maybe `universal`